storage/mysql/log_storage.go

```diff
@@ -474,7 +487,11 @@ func (t *logTX) UpdateSequencedLeaves(leaves []trillian.LogLeaf) error {
 			return errors.New("Sequenced leaf has incorrect hash size")
 		}
 
-		_, err := t.tx.Exec(insertSequencedLeafSQL, t.ls.logID.TreeID, []byte(leaf.LeafHash),
+		// Recompute the raw leaf hash, could be passed around in the leaf structure but
```

Comment:
Working on this now. I'm going to take the hit now and rename everything in Leaf / Logleaf end to end so it's always consistent.